tests/baselines/reference/user/lodash.log

```diff
@@ -76,6 +76,7 @@ node_modules/lodash/_baseUniq.js(30,5): error TS2322: Type '(array?: any[] | und
 node_modules/lodash/_baseUniq.js(33,33): error TS2554: Expected 0 arguments, but got 1.
 node_modules/lodash/_baseUniq.js(39,5): error TS2322: Type 'SetCache' is not assignable to type 'any[]'.
 node_modules/lodash/_baseUniq.js(62,15): error TS2554: Expected 2 arguments, but got 3.
+node_modules/lodash/_baseWrapperValue.js(18,21): error TS2339: Property 'value' does not exist on type 'LazyWrapper'.
```

Comment:
better narrowing again in lodash; these are, again, correct errors since the properties we can't find are initialised dynamically.